src/utils.js

```diff
@@ -10,16 +10,47 @@ export const formatDate = date => format(date, 'YYYY-MM-DD');
 // :: String -> ((Date | String | Int) -> Int -> Date)
 export const getSubtractionFn = (unit) => {
   if (unit === 'year') return subYears;
-  if (unit === 'month') return subMonths;
   if (unit === 'quarter') return (date, nb) => subMonths(date, nb * 3);
+  if (unit === 'month') return subMonths;
   if (unit === 'week') return subWeeks;
   return subDays;
 };
 
+// Parse timezone string and returns its hours equivalent (`+08:00` -> 8)
+// :: String -> Float
+export const parseTimezone = (tz) => {
+  const [chunk1, chunk2] = tz.split(':');
+  const isPositive = chunk1[0] === '+';
+  const hours = parseInt(chunk1.slice(1), 10);
+  const minutes = parseInt(chunk2, 10);
+  const result = hours + (minutes / 60);
+  if (isPositive) return result;
+  return -result;
+};
+
+// The offset can be either in hours or in seconds. Or it can be a string.
+// Returns the offset in miliseconds
+// :: (Int | String) -> Int
+export const normalizeOffset = (offset) => {
+  // It's not a number: we assume it's in the shape of `+08:00` and parse it
+  if (typeof offset !== 'number') return parseTimezone(offset) * 3600 * 1000;
+  // Offset in seconds: we convert to miliseconds
+  if (Math.abs(offset) > 15) return offset * 1000;
```

Comment:
what about negative values ?
Also not a big fan of comments, you could have a method name that says what it does instead: `isSeconds` for instance. 
IMO comments too often get outdated and become noise, not mentioning typos.

Reply:
My bad, realise you used `abs`

Reply:
I usually never leave comments inside of functions, but here is an exception: this function, by essence, does some serious implicit magic. Like, it gets a number, then kind of guuesses whether it's seconds or hours, then acts on it...
I felt like I needed to comment this, otherwise people can't guess what it's doing. An `isSeconds` function would also solve that problem, but then this function would be very misleading bc it's not generic at all, it's guessing if it's seconds in a very specific context, is not reusable, etc.
Like, when given a number, you can't just guess whether it's seconds or hours, period. So it'd have to be something like `guessIfSecondsWhenOffsetValue` but...might as well not make it a function honestly.